web_src/js/components/DiffCommitSelector.vue

```diff
@@ -32,6 +32,7 @@ export default defineComponent({
       locale: {
         filter_changes_by_commit: el.getAttribute('data-filter_changes_by_commit'),
       } as Record<string, string>,
+      merge_base: el.getAttribute('data-merge-base'),
```

Comment:
I don't think it is a proper naming style.

<img width="1508" height="510" alt="Image" src="https://github.com/user-attachments/assets/dc90188e-f50a-4fc8-91aa-c251b8fcb23f" />

Reply:
2ac3e8b29c